mteb/model_meta.py

```diff
@@ -80,6 +81,7 @@ class ModelMeta(BaseModel):
 
     name: str | None
     revision: str | None
+    tokenizer_name: str | None
```

Comment:
You should pass `tokenizer_name` to `loader` like this 
`loader=partial(OpenAIWrapper, model_name="text-embedding-3-large", tokenizer_name="tokenizer", max_tokens=8192),`

Reply:
Okay, I'll revise this

Reply:
Revised, and I'll do one more evaluation to see if it works well